src/components/Profile/ProfileContainer.js

```diff
@@ -4,13 +4,15 @@ import { withRouter } from 'react-router-dom';
 import { connect } from 'react-redux';
 import { compose } from 'redux';
 
-import { getUserProfile, getStatus, updateStatus } from '../../redux/actions/thunks';
+import {
+  getUserProfile, getStatus, updateStatus, savePhoto, saveProfile,
```

Comment:
In case the linter doesn't mind it - it's better to keep the imported things each on a new line.
E.g. 
`import {`
`    getUserProfile,`
`    getStatus,`
`...`
`} from '..url'`

Reply:
A general rule of readability - if things are more than 3, keep them on a new line.

Reply:
/fxd